global.variables.yml

```diff
@@ -24,8 +24,8 @@ variables:
   ######################################
 
   bicepRegistryDoPublish: true # Set to true, if you would like to publish module templates to a bicep registry
-  bicepRegistryName: adpsxxazacrx001 # The name of the bicep registry (ACR) to publish to. If it does not exist, it will be created.
-  bicepRegistryRGName: 'artifacts-rg' # The resource group that hosts the private bicep registry (ACR)
+  bicepRegistryName: msxbicep # The name of the bicep registry (ACR) to publish to. If it does not exist, it will be created.
```

Comment:
```suggestion
  bicepRegistryName: adpsxxazacrx001 # The name of the bicep registry (ACR) to publish to. If it does not exist, it will be created.
```